assets/src/edit-story/components/keyboardShortcutsMenu/getKeyboardShortcuts.js

```diff
@@ -141,10 +163,7 @@ function getKeyboardShortcuts() {
             shortcut: [cmdOrCtrl, 'K'],
           },
           {
-            label: __(
-              'Disable snapping during move/rotate/resize',
-              'web-stories'
-            ),
+            label: __('Disable snapping during drag', 'web-stories'),
```

Comment:
Hmm, just `drag` doesn't seem precise enough to reflect that it happens when resizing as well. Thoughts?

Reply:
It was mostly for visual reasons, as the previous version looked crappy IMO. The shortcuts take more space now with the longer shortcuts for move elements in layer ordering, so there's less room for the descriptions.

@o-fernandez, what do you think?

Old text before:
![Screen Shot 2020-09-17 at 09 07 40](https://user-images.githubusercontent.com/637548/93474386-44847a80-f8c5-11ea-819e-b7161cdd7123.png)


Old text now:
![Screen Shot 2020-09-17 at 09 06 21](https://user-images.githubusercontent.com/637548/93474338-346c9b00-f8c5-11ea-95ae-9dd68b295bef.png)


New text:
![Screen Shot 2020-09-17 at 09 06 50](https://user-images.githubusercontent.com/637548/93474327-30d91400-f8c5-11ea-9ffb-be09fb8d27b9.png)

Reply:
And yes, we could reduce the space used for the shortcuts by having separate entries for move element up/down and front/back, but then the list becomes longer and that also looked back - especially compared to the left hand side:

Separate entries:
![Screen Shot 2020-09-17 at 09 34 43](https://user-images.githubusercontent.com/637548/93477482-1143ea80-f8c9-11ea-8e3c-1a60c6a0418b.png)

Compact entries:
![Screen Shot 2020-09-17 at 09 32 27](https://user-images.githubusercontent.com/637548/93477505-19038f00-f8c9-11ea-8fcc-89cbda2f232d.png)

Reply:
I think the compact version loooks good to me.  

But, we could also take the layer shortcuts out of the `Element` section, and create a new `Layer` section that only has those shortcuts.  Then you could update the shortcut render logic so the left column has `Basics`, `Text`, `Layers`  shortcuts and the right side only has `Element`.  Maybe we can add a `col` prop to each section with values left or right (if it's missing it assumes left), and then the columns could be easily rearranged by changing the col property per section 🤔.

Reply:
I like compact. In terms of language 
**"Disable snapping while moving"** 
seems more natural to me.